docs-go/get-started-go.md

```diff
@@ -0,0 +1,148 @@
+# Get started with Genkit using Go
+
+To get started with Firebase Genkit, install the Genkit CLI and run
+`genkit init` in a Go project. The rest of this page shows you how.
+
+## Requirements
+
+- Go 1.22 or later
+- Node.js 20 or later (for the Genkit CLI and UI)
+
+## Procedure
+
+1.  Install the Genkit CLI by running the following command:
```

Comment:
Assume readers have no trace of any node software on their machine, and know nothing about the node ecosystem.

Someone suggested starting by installing nvm and going from there.

If feasible, there should be two sets of instructions: one for people with root access who can do a global install, and also a way to install for just the current user.